docs/node/daemon/config/settings.mdx

```diff
@@ -286,17 +313,17 @@ The structure of `config.toml` is hierarchical, using sections and subsections.
 :::note Translation rules
 ```toml title=config.toml
 [section]
-   field1 = "val1"
+field = "val"
+
 [section.subsection]
-   field2 = "val2"
+other_field = "val"
 ```
 
-The corresponding command line flags and environment variables for `field1` and
-`field2` are as follows:
+The corresponding command line flags and environment variables for `field` and
+`other_field` are as follows:
 
 | Flag                               | Environment Variable                   |
 |------------------------------------|----------------------------------------|
-| `--section.field1`                 | `KWILD_SECTION_FIELD1`                 |
-| `--section.subsection.field2-name` | `KWILD_SECTION_SUBSECTION_FIELD2_NAME` |
-
-:::
\ No newline at end of file
+| `--section.field`                  | `KWILD_SECTION_FIELD`                  |
+| `--section.subsection.other-field` | `KWILD_SECTION_SUBSECTION_OTHER_FIELD` |
```

Comment:
I'm pretty sure `field2-name` was wrong here.  I'm now demoing with an entirely different field name to show the dash/slash translation.